packages/commonwealth/server/api/external-router.ts

```diff
@@ -63,6 +70,10 @@ const api = {
   getThreads: trpc.query(Thread.GetThreads, trpc.Tag.Thread, {
     forceSecure: true,
   }),
+  getAllContests,
```

Comment:
if you are exposing a non-secure query, you have to pass forceSecure to enable auth